src/cfml/system/services/EndpointService.cfc

```diff
@@ -92,6 +92,15 @@ component accessors="true" singleton {
 				package : arguments.ID,
 				ID : endpointName & ':' & arguments.ID
 			};
+		// Is it a Gist user/gistID?
```

Comment:
I added this check because the NPM documentation talks about the username being optional and removing it before saving it to the `package.json file`. 

> The GitHub username associated with the gist is optional and will not be saved in package.json.

It is checking to see the user passed `username/gistID` and removing the username from the ID. I should have made it more clear in my comment as to why it is there. I can take this out if you are ok with just leaving `username/gistID` as the package ID.

Reply:
I would leave it as-is.  It's not really the job of this method to be changing what the user provides, just detecting what the proper endpoint is. If the endpoint wants to ignore part of the ID, then that's up to it.